### PR TITLE
Make sdk_xcode_harness similar to hello_services

### PR DESCRIPTION
**Pull request by abarth**
This patch makes the sdk_xcode_harness work the same way as
hello_services. The goal is to eventually make these the same, but this
is just a first step.

### CODE REVIEW

**Comment by abarth** [comment]
@chinmaygarde @tvolkert 

Landing this patch will require a bunch of other changes (e.g., to the buildbot and to flutter_tools), but I wanted to post this patch to get feedback on the approach.

**Comment by abarth** [comment]
flutter_tools changes are in https://github.com/flutter/flutter/pull/4268

File: sky/build/sky_ios_sdk.gni [hunk 1]
```diff
@@ -13,74 +13,17 @@ template("sky_ios_sdk") {
 
   sdk_name = invoker.sdk_name
   sdk_dir = "$root_out_dir/$sdk_name"
-  tools_dir = "Tools/common"
-
-  action("copy_flutter_framework") {
-    stamp_file = "$root_build_dir/copy_flutter_framework.stamp"
-    script = "//sky/tools/copy_dir.py"
```
Comment (tvolkert) [inline sky/build/sky_ios_sdk.gni]: Looks like //sky/tools/copy_dir.py can be removed now

File: sky/build/sdk_xcode_harness/Flutter/Flutter.xcconfig [hunk 2]
```diff
@@ -0,0 +1 @@
+#include "Generated.xcconfig"
```
Comment (tvolkert) [inline sky/build/sdk_xcode_harness/Flutter/Flutter.xcconfig]: Does Flutter.xcconfig provide any value anymore?

Comment (abarth) [inline sky/build/sdk_xcode_harness/Flutter/Flutter.xcconfig]: The value is that it's checked into source control and references a file that's outside source control.  The Xcode project can reference Flutter.xcconfig without breaking even if Generated.xcconfig doesn't exist.

**Comment by tvolkert** [comment]
lgtm

**Comment by abarth** [comment]
I tested this locally.  Going to try landing it.

**Comment by chinmaygarde** [comment]
How did you test this locally? This gets rid of the step that creates the `FlutterXcode.zip` that is used by `--local-engine`.

**Comment by abarth** [comment]
I modified the bin/cache directory directly to contain the new bits.

**Comment by chinmaygarde** [comment]
I see. Ok. Will add back an archive step that recreates a similar ZIP.

**Comment by abarth** [comment]
We might not need that.  The Xcode project is now independent of the build configuration because it doesn't contain any binaries.  We should be headed towards the Xcode project being stable, which means we could just skip the step of finding the zip file.

**Comment by abarth** [comment]
I think the end-state here is for the Xcode project to be generated by `flutter create`, like the Info.plist.  We don't need to do that right away, but I think that's what we're going to want in the end.

**Comment by tvolkert** [comment]
+1

**Pull request merged**